src/main/java/bio/terra/cli/apps/interfaces/Supported.java

```diff
@@ -0,0 +1,21 @@
+package bio.terra.cli.apps.interfaces;
+
+import bio.terra.cli.context.GlobalContext;
+import bio.terra.cli.context.WorkspaceContext;
+
+/** Classes that implement this interface can be enabled with the 'terra app enable __' command. */
+public interface Supported {
+  /** Set the global and workspace context properties. */
+  void setContext(GlobalContext globalContext, WorkspaceContext workspaceContext);
+
+  /**
+   * This enum specifies the list returned by the 'terra app list' command. Classes that implement
+   * this interface should also be added to this enum.
```

Comment:
That was the Supported interface. List interface seemed like a confusing name, since it didn't have anything to do with a Java list. Supported is also the word used in the user-facing output around `terra app list`.